CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- ⚡️ The GitHub CI now runs on Debian Buster and produces Debian artifacts.
+  Similarly, the Docker images we provide on
+  [dockerhub](https://hub.docker.com/r/tenzir/vast) use Debian Buster as
+  base-image.
+  [#1294](https://github.com/tenzir/vast/pull/1294)
```

Comment:
Could we add sentence fragment about what we had before? Not everyone might be aware of that.

Reply:
Thanks, reads much more understandable now!